tests/test_web_exceptions.py

```diff
@@ -9,9 +10,9 @@
 
 
 def test_all_http_exceptions_exported() -> None:
-    assert 'HTTPException' in web.__all__
+    assert "HTTPException" in web.__all__
```

Comment:
I'm curious why did you replace all quote symbols?
The change is not related to the PR purpose.

Reply:
Oops, sorry me. It's black.

Reply:
Let's do black reformatting in another PR.
Since 19.10 it can be applied to Python 3.5 compatible code.
The trick is that blackifying should be done on all supported branches at the same time; otherwise backporting becomes virtually impossible.